src/github.com/getlantern/lantern-mobile/interface.go

```diff
@@ -2,7 +2,9 @@ package client
 
 // StopClientProxy stops the proxy.
 func StopClientProxy() error {
-	defaultClient.stop()
+	if err := defaultClient.stop(); err != nil {
+		log.Debugf("Unable to stop client: %v", err)
```

Comment:
I'll set it as an error too for correctness. But it probably won't log to loggly, just locally.